chart/examples/k3s-hosted.yaml

```diff
@@ -0,0 +1,83 @@
+# Hosted Config (eg. for k3s)
+# --------------------------------
+#
+# This config can be used to run Browsertrix Cloud hosted on a remote server.
+# This requires setting the hostname and signing hostname.
+#
+#
+# The `ingress.host` and `ingress.cert_email` and optional `signer.host` must be set below.
+#
+# k3s
+# ---
+#
+# One way to deploy this is with k3s, which will require the following changes:
+#
+# Make sure to disable traefik on your k3s cluster by adding `--no-deploy traefik` to the systemd unit that starts k3s _before_ starting your server. One way to check that traefik is not running your cluster is with `kubectl get deployments -n kube-system` and see if traefik shows up in the output
+#
+# Once traefik has been disabled, you must install `nginx-ingress` which can be installed by: 
+# helm upgrade --install nginx ingress-nginx/ingress-nginx -n ingress-nginx --create-namespace
+#
+# You'll also need to install cert-manager. Browsertrix cloud recommends installing cert-manager through jetpack with the following command: 
+#
+# helm repo add jetstack https://charts.jetstack.io
```

Comment:
I wonder if this should actually be a script that we add, so easier to run?

Reply:
Similar to `eck_install`? 
We do provide the ansible playbook for an "easier" install